src/maingoblint.ml

```diff
@@ -243,6 +243,8 @@ let preprocess_files () =
   if get_bool "custom_libc" then
     cFileNames := find_custom_include "lib.c" :: !cFileNames;
 
+  cFileNames := find_custom_include "stdlib.c" :: !cFileNames;
```

Comment:
This makes me wonder what the point of `custom_libc` really is. The only place were AFAIK it is used is `pfscan` in the bench repository to have an overly simple stub for `ftw` (which isn't even the C standard library, but some Unix one).

I didn't want to add `qsort` and `bsearch` in `lib.c` because `custom_libc` is disabled by default, but we should still be sound for `qsort` and `bsearch` even without enabling some custom option.

These now additions make the meaning of `custom_libc` confusing though, because this is another custom libc thing which actually doesn't obey the option.

Reply:
`ftw` is another of those functions which uses a callback though, so maybe it should be just in `stdlib.c` always anyway. Or possibly another file `ftw.c` which is added by default to be more clear that it isn't C standard library.

Reply:
Since it was just specific to pfscan, we decided to move its stub to the bench repository: https://github.com/goblint/bench/commit/c31ef8287e2b77cdbe0c2ba577e29b0bfa7cb4d0. Now `custom_libc` option is removed here and the stdlib stubs are always used.